kie-remote/kie-remote-client/src/test/java/org/kie/services/client/builder/RemoteClientBuilderTest.java

```diff
@@ -760,4 +760,25 @@ public void execptionWhenUserIdDoesNotMatchAuthUserId() throws Exception {
         }
     }
 
+
+    @Test
+    public void setHeaderFields() throws Exception {
+        String authUser = "user";
+        RuntimeEngine runtimeEngine =
+                RemoteRuntimeEngineFactory.newRestBuilder()
+                .addUserName(authUser)
+                .addPassword("pass")
+                .addUrl(new URL("http://localhost:8080/business-central"))
+                .addHeader("HEADER-NAME", "header-value")
+                .build();
+
+        try {
+            runtimeEngine.getTaskService().claim(23l, "user");
```

Comment:
Correct me if I'm wrong, but didn't you declare ```authUser``` above for the exact purpose of being able to reuse it here?